.github/workflows/ci.yml

```
@@ -0,0 +1,96 @@
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      mongodb:
+        image: mongo:latest
+        ports:
+          - 27017:27017
+        options: --health-cmd "mongosh --eval 'db.runCommand(\"ping\").ok'" --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)